From 4edfee6bd2be248de18d0c865be87cb0f72ddd28 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 16 Aug 2019 06:30:35 -0400
Subject: [PATCH] limit product name less then 30 chars

Previously 40 is still too long, which broke gui window layout

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/storage/devicetree/root.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/modules/storage/devicetree/root.py b/pyanaconda/modules/storage/devicetree/root.py
index 1af402e2d..b375e9164 100644
--- a/pyanaconda/modules/storage/devicetree/root.py
+++ b/pyanaconda/modules/storage/devicetree/root.py
@@ -174,7 +174,7 @@ def get_release_string(chroot):
     except ValueError:
         pass
 
-    return rel_arch, rel_name, rel_ver
+    return rel_arch, rel_name[:30], rel_ver
 
 
 def _release_from_redhat_release(fn):
-- 
2.8.1

